frontend/src/components/TradeBox.js

```diff
@@ -1409,7 +1335,7 @@ class TradeBox extends Component {
         enterTouchDelay={0}
         title={
           <Trans i18nKey='open_dispute'>
-            To open a dispute you need to wait{' '}
+            To open a dispute you need to wait
```

Comment:
Not a translation?

Reply:
It is a translation, in this case we <Trans/> component instead of t(). We use the <Trans/> here because this translation has a children component <Countdown...>.

When running `npm run format` a few days ago the `{' '}` was introduced, I am not sure why. It made the <Countdown> component dissapear from the translations, as they were tagged with <1></1> (that used signal where to render the <Coundown/> in the translation) move and instead render the empty string ' '. 
https://github.com/Reckless-Satoshi/robosats/blob/cd8b607891a786c77986ffee45dbd75f51610a01/frontend/static/locales/en.json#L461

Reply:
(sorry for mixing this bug fix in this PR... totally unrelated, it was bugging me!)